IrisHub/SpaceBar
# Sam/world boundaries

### Description of Changes
Introduce world boundaries with addition of boundaries.tsx and modification of floor.tsx. 
Refactor shapes library for clarity, introduce new props to make shapes translucent.
Introduced a changelog for us to maintain over time.

![image](https://user-images.githubusercontent.com/31233283/163704133-e2eb3817-4e23-4d5f-ba75-3697601e91b9.png)


### Link to Notion ticket
https://www.notion.so/irisapp/Add-a-world-boundary-and-define-behavior-at-end-of-the-world-4971961ebda04cc397830963ac458803

### Testing Procedure
Tested each prop in boundaries & shapes lib locally in combination with other props to ensure expected behavior. Modified world constants for height and width to ensure that boundaries resize correctly.
